goreleaser.yml

```diff
@@ -0,0 +1,64 @@
+project_name: miacli
+before:
+  hooks:
+    - go mod download
+    - go generate ./...
+release:
+  prerelease: false
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - 386
+      - amd64
+      - arm64
+      - arm
+    goarm:
+      - 7
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X github.com/mia-platform/miacli/cmd.version={{.Version}} -X github.com/mia-platform/miacli/cmd.commit={{.Commit}} -X github.com/mia-platform/miacli/cmd.date={{.Date}}
+signs:
+  - artifacts: none
+
+archives:
+  - name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}"
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      bit: Arm
+      bitv6: Arm6
+      bitv7: Arm7
+      386: i386
+      amd64: x86_64
+checksum:
+  name_template: "checksums.txt"
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
+
+# Homebrews
+brews:
+  - name: miacli
+    github:
+      owner: mia-platform
+      name: homebrew-tap
+    commit_author:
+      name: thobianchi
+      email: thomas.bianchi8@gmail.com
+    folder: Formula
+    homepage: https://mia-platform.eu
+    description: miactl is the cli of the mia-platform DevOps Console
+    test: |
+      system "examplecli help"
```

Comment:
Here should be the example to test after brew installation?

Reply:
yes, here some docs https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula
basically:
> Add a valid test to the test do block of the formula. This will be run by brew test foo and the Brew Test Bot.

For a first implementation I think is sufficient that `miactl help` gives no errors.

PS
replaced `examplecli` with `miactl`